contrib/systemd/user-network.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=vsock_network
+After=NetworkManager.service
+
+[Service]
+[Service]
+Environment=GV_TAP_DEVICE="vsock0"
+Environment=GV_VSOCK_PORT="1234"
+EnvironmentFile=-/etc/sysconfig/gv-user-network
+ExecStart=/usr/libexec/podman/vm -preexisting -iface $GV_TAP_DEVICE -url vsock://2:$GV_VSOCK_PORT/connect
```

Comment:
For some reason, systemd wants `{}` around the env var names, otherwise it won't expand `GV_VSOCK_PORT`: `ExecStart=/usr/libexec/podman/vm -preexisting -iface ${GV_TAP_DEVICE} -url vsock://2:${GV_VSOCK_PORT}/connect`